### ISSUE TITLE
Questions about how to solve the extremely unbalanced data

### ISSUE THREAD

**Issue by wishx98** [songtaohe/LaneExtraction#6]
Hello, Songtao:
        When I was reproducing your program laneAndDirectionExtraction in PyTorch, I encountered the extremely imbalanced data that made it difficult for the network to learn and extract roads (roads only account for 0.37% in the training set). Even if I followed the celoss and dice loss in the paper and this program. Could you share how you solve this problem? I would be very grateful if you could reply!

**Comment by a4andromeda** [comment]
@StoneSWX I run into the same problem with the reference code. Did you ever solve it ?
